src/components/atoms/button/button.tsx

```diff
@@ -9,18 +8,20 @@ type buttonProps = {
   isLoading:boolean,
   isDisabled:boolean,
   isPrimary:boolean,
+  onClickHandler(event:any): any;
```

Comment:
Ideally we would want all action handlers to have a void return value since their job is to just handle actions for the most part. Would you agree?

Reply:
> Ideally we would want all action handlers to have a void return value since their job is to just handle actions for the most part. Would you agree?

yeah i agree will change this on void.